### PR TITLE
Add network module

### PR DESCRIPTION
**Pull request by ZakariaRidouh**
We can write a component for this later that we can then include in the Movie and Show components.

### CODE REVIEW

**Comment by amilajack** [comment]
Ideally, this would return a promise that resolves to an object. Also I think the object should be in the format of something like this:

```js
{
  download: {
    kbps: 100,
    mbps: 200
  },
  {
    upload: {
      // ...
    }
  }
}
```

**Comment by amilajack** [comment]
Still linter errors

**Comment by ZakariaRidouh** [comment]
Haha I suggest you squash this PR 😄

**Comment by amilajack** [comment]
I'm probably going to squash the release branch

**Comment by amilajack** [comment]
Also can you write a test for this? That would be really nice. Here's the steps:

```md
1. Add file `utils.spec.js` in `tests`
2. Import the module
3. Write a test. I would recommend using the other tests as examples. The tests use mocha and chai for assertion.
```

File: test/utils.spec.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import { expect } from 'chai';
+
+
+describe('Network', () => {
+  describe('download', () => {
+    it('should respond with the download speed', (done) => {
+      expect(getDownloadSpeed()).to.be.a('number');
+      done();
+    });
+  });
```
Comment (amilajack) [inline test/utils.spec.js]: There should be a new line after this

File: app/utils/network.js [hunk 2]
```diff
@@ -6,16 +6,27 @@ const test = speedTest({
...
+  });
+}
+
+export function getUploadSpeed() {
+  test.on('data', (data) => {
     console.log('Upload Speed: ${data.speeds.upload} Mb/s');
-    return data; // the whole object for now
+    return data.speeds.upload;
   });
   test.on('error', (err) => {
```
Comment (amilajack) [inline app/utils/network.js]: New line

File: app/utils/network.js [hunk 3]
```diff
@@ -6,16 +6,27 @@ const test = speedTest({
   maxTime: 5000
 });
 
-export default function getSpeed() {
+export function getDownloadSpeed() {
   test.on('data', (data) => {
     console.log('Download Speed:  ${data.speeds.download} Mb/s');
+    return data.speeds.download;
+  });
```
Comment (amilajack) [inline app/utils/network.js]: New line after this

**Comment by ZakariaRidouh** [comment]
@amilajack the tests should be working now 😄

**Comment by amilajack** [comment]
This project tries to maximize the use of promises so that async/await or promises can be used. Ideally, the module can be used like so:


```js
const speed = await getUploadSpeed()
```

**Comment by amilajack** [comment]
Also test that the api is returning a promise:

```js
expect(getUploadSpeed).to.be.a('promise')
```

**Comment by amilajack** [comment]
Merged!

**Pull request merged**